chapter_recurrent-neural-networks/sequence_vn.md

```diff
@@ -559,13 +559,13 @@ Do đó, nếu bạn có một kiểu dữ liệu chuỗi thời gian, hãy luô
 -->
 
 1. Hãy cải thiện mô hình nói trên bằng cách
-    * Kết hợp nhiều hơn 4 mẫu quan sát trong quá khứ? Bạn thực sự cần bao nhiêu mẫu quan sát?
-    * Bạn sẽ cần bao nhiêu mẫu nếu dữ liệu không có nhiễu? Gợi ý: bạn có thể viết $\sin$ và $\cos$ dưới dạng phương trình vi phân.
-    * Có thể kết hợp các đặc trưng cũ hơn trong khi đảm bảo tổng số đặc trưng là không đổi không? Điều này có cải thiện độ chính xác không? Tại sao?
-    * Thay đổi cấu trúc mạng nơ-ron và xem những gì sẽ xảy ra.
-2. Nếu một nhà đầu tư muốn tìm một mã chứng khoán tốt để mua. Cô ta sẽ nhìn vào lợi nhuận trong quá khứ để quyết định mã nào có khả năng sinh lời. Điều gì có thể khiến chiến lược này trở thành sai lầm?
-3. Liệu có thể áp dụng quan hệ nhân quả cho dữ liệu văn bản được không? Nếu có thì ở mức độ nào?
-4. Hãy cho một ví dụ khi một mô hình tự hồi quy ẩn có thể cần được dùng để nắm bắt động lực của dữ liệu.
+    * Kết hợp nhiều hơn 4 mẫu quan sát trong quá khứ? Bao nhiêu mẫu quan sát là thực sự cần thiết? 
+    * Bạn sẽ cần bao nhiêu mẫu nếu dữ liệu không có nhiễu? Gợi ý: bạn có thể viết $\sin$ và $\cos$ dưới dạng phương trình vi phân. 
+    * Có thể kết hợp các đặc trưng cũ hơn trong khi đảm bảo tổng số đặc trưng là không đổi không? Điều này có cải thiện độ chính xác không? Tại sao? 
+    * Thay đổi cấu trúc mạng nơ-ron và xem những gì sẽ xảy ra. 
+2. Nếu một nhà đầu tư muốn tìm một mã chứng khoán tốt để mua. Cô ta sẽ nhìn vào lợi nhuận trong quá khứ để quyết định mã nào có khả năng sinh lời. Điều gì có thể khiến chiến lược này trở thành sai lầm? 
+3. Liệu có thể áp dụng quan hệ nhân quả cho dữ liệu văn bản được không? Nếu có thì ở mức độ nào? 
+4. Hãy cho một ví dụ khi mô hình tự hồi quy tiềm ẩn có thể cần được dùng để nắm bắt tính động của dữ liệu.
```

Comment:
có lý do gì để thay `động lực` không nhỉ, trong các phần trước mình toàn xài nó cho từ `dynamic`.
```suggestion
4. Hãy cho một ví dụ khi mô hình tự hồi quy tiềm ẩn có thể cần được dùng để nắm bắt động lực của dữ liệu.
```